packages/shared/src/forms/__tests__/SliderLabels.test.js

```diff
@@ -0,0 +1,179 @@
+// @flow strict
+
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+import ShallowRenderer from 'react-test-renderer/shallow';
+
+import Translation from '../../Translation';
+import SliderLabels from '../SliderLabels';
+
+const shallowRenderer = new ShallowRenderer();
+
+it('one label', () => {
```

Comment:
```suggestion
test('one label', () => {
```

Reply:
I like `it`, but I will change it to `it('renders with one label)`